image.go

```diff
@@ -154,6 +157,7 @@ func (i *containerImageRef) NewImageSource(ctx context.Context, sc *types.System
 		return nil, err
 	}
 	dimage.Created = created
+	dimage.Config.OnBuild = i.onBuild
```

Comment:
This was my other attempt to get OnBuild into the destination image, but it too doesn't seem to take.

Reply:
This shouldn't be necessary -- we default to preserving any value that's been set there when reading the image in, and saving the value that's in the structure when writing the config blob as part of committing the image.

Reply:
oops, forgot to note that I've removed this.